simple-wp-helpdesk/assets/swh-frontend.css

```
@@ -1,291 +1,393 @@
+	align-items: center;
+	gap: 5px;
+	margin-right: var( --swh-space-sm );
+	text-decoration: underline;
+	color: var( --swh-color-primary );
+	font-size: 13px;
+}
+
+.swh-attachment-item .swh-file-icon {
+	width: 14px;
+	height: 14px;
+	fill: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)